lib/win.js

```diff
@@ -49,6 +49,24 @@ var WinBuilder = {
       options.log('makensis error: ' + data);
     });
 
+    makensis.on( 'error', function ( err ) {
+      if (err.code === 'ENOENT') {
+        options.log( '- makensis not found.');
+        options.log( '  Please install "nullsoft scriptable install system"' );
+        switch(process.platform) {
+          case 'darwin':
+          case 'win32':
+            options.log( '  http://nsis.sourceforge.net/Main_Page' );
+            break;
+          case 'linux':
+            options.log( '  $ sudo apt-get nsis' );
+        }
+        options.log();
```

Comment:
I added the log so there was a space between the info printed here and the error that gets printed out next. The actual output looks like this:

```
> electron-builder ./release/win32/electron-updater-sample-win32 --platform=win32 --config=config.json --out=./release/win32

- Starting build for ´win32´ -
- makensis not found.
  Please install "nullsoft scriptable install system"
  http://nsis.sourceforge.net/Download

c:\Users\justin\projects\github\electron-builder\cli.js:30
    throw error;
          ^
```

Your call, I can leave it or remove it however you want.